cmake/Modules/LocalRav1e.cmake

```diff
@@ -1,6 +1,6 @@
 set(AVIF_RAV1E_GIT_TAG v0.7.1)
 set(AVIF_CORROSION_GIT_TAG v0.5.1)
-set(AVIF_CARGOC_GIT_TAG v0.10.11)
+set(AVIF_CARGOC_GIT_TAG 8c5b7af3d6edb6d99f7bffcc94adf550cfee65b3)
```

Comment:
When I wrote this pull request to revert the temporaty cargo.patch, I didn't realize we were checking out a release tag of cargo-c. Let's wait for a new cargo-c release that contains this commit.

Reply:
This commit is now available as cargo-c release tag v0.10.12.